
The usual fixes to make the code leagal C for gcc-3.4.

  - Rene Rebe <rene@rocklinux.org>

--- ./apmd.c.vanilla	2004-05-26 17:53:37.000000000 +0200
+++ ./apmd.c	2004-05-26 17:56:55.000000000 +0200
@@ -71,8 +71,8 @@
 
 
 #ifdef APM_TRACE
-#define ADEBUG(lev,args...) \
-    if (APM_TRACE>=lev) syslog(LOG_DEBUG, __FUNCTION__ ": " args)
+#define ADEBUG(lev, text, args...) \
+    if (APM_TRACE>=lev) syslog(LOG_DEBUG, "%s : " text, __FUNCTION__, args)
 #else
 #define ADEBUG(args...)
 #endif
